SECURITY.md

```diff
@@ -1,41 +1,56 @@
+to artifacts, attestations).
 
 ## Reporting a vulnerability
 
 **Please do not file a public issue or PR mentioning the vulnerability.**
 
-Please visit [https://stacks.org/security](https://stacks.org/security) for the
-most up-to-date information on our security policy.
+If you have identified a vulnerability, please:
 
-You may also contact us with any questions or to report a vulnerability using
-the following key(s):
+1. E-mail us your findings at `security@bitcoinl2labs.com`.
+1. Encrypt your findings using our PGP key, provided below.
 
-| Name                | Fingerprint                                       |
-| ------------------- | ------------------------------------------------- |
-| security@stacks.org | ABA3 7FA3 6DBB A591 B0E5 5949 0E94 D065 B32E C7E6 |
+| Name                       | Fingerprint                                         |
+| -------------------------- | --------------------------------------------------- |
+| security@bitcoinl2labs.com | `DFD9 33A6 C455 0ED4 C046 7133 F78A DC4F 9F65 58DD` |
 
-```
+```txt
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mDMEZrJ2wBYJKwYBBAHaRw8BAQdADVWSZGbVgc0SE8XmXkRonl85wXrPHkl9bN0B
-jKFBIRS0KXNlY3VyaXR5QHN0YWNrcy5vcmcgPHNlY3VyaXR5QHN0YWNrcy5vcmc+
-iJAEExYIADgWIQSro3+jbbulkbDlWUkOlNBlsy7H5gUCZrJ2wAIbAwULCQgHAgYV
-CgkICwIEFgIDAQIeAQIXgAAKCRAOlNBlsy7H5tznAQC6iKqtjCqn2RjtCkr2V6xe
-kCe92RfwWsG0415jVpVlDgEA350TCqIT1Jwyqz2aNT2TQ9F6fyKzAiNpLVRImOLH
-4Aq4OARmsnbAEgorBgEEAZdVAQUBAQdAvwusRitvUX9hSC8NKS48VTT3LVvZvn87
-JQXRc2CngAEDAQgHiHgEGBYIACAWIQSro3+jbbulkbDlWUkOlNBlsy7H5gUCZrJ2
-wAIbDAAKCRAOlNBlsy7H5oCNAQDae9VhB98HMOvZ99ZuSEyLqXxKjK7xT2P0y1Tm
-GuUnNAEAhI+1BjFvO/Hy50DcZTmHWvHJ6/dzibw5Ah+oE458IQo=
-=yhSO
+mDMEZ+0wkRYJKwYBBAHaRw8BAQdAZd3UBaYBvYr3wzmrvhe/gZ8J53CO7eeQA5Yo
+XrgpDK60N3NlY3VyaXR5QGJpdGNvaW5sMmxhYnMuY29tIDxzZWN1cml0eUBiaXRj
+b2lubDJsYWJzLmNvbT6IlAQTFgoAPBYhBN/ZM6bEVQ7UwEZxM/eK3E+fZVjdBQJn
+7TCRAhsDBQkB4TOABAsJCAcEFQoJCAUWAgMBAAIeBQIXgAAKCRD3itxPn2VY3RkN
+AP46xrKE+BPoWlIeaPdoiQ9K5qIMW1p2yUm80mHhM9kXrgEA3BR6Le6ryQISjfSH
+TpdJGxc92QQsFmy05eTDjggC4wG4OARn7TCREgorBgEEAZdVAQUBAQdAIgllD3+6
+zq9SemqJ/bIyseu3RBGPLmeayrxtiwiy1S8DAQgHiH4EGBYKACYWIQTf2TOmxFUO
+1MBGcTP3itxPn2VY3QUCZ+0wkQIbDAUJAeEzgAAKCRD3itxPn2VY3SkXAP41ekdx
+p4m+39c5PdaprZrz7su8EJ8SOwiuDf6QQI9hrwD/UmN6EES8THy8xN5vj5WMUgXF
+YlmGr2MoyB92KmX3gwc=
+=Ou/9
 -----END PGP PUBLIC KEY BLOCK-----
 ```
 
 ## ImmuneFi
 
-The Stacks Foundation has partnered with [ImmuneFi](https://immunefi.com/) to
-reward honest researchers who find and responsibly disclose security
-vulnerabilities in our critical code. Bounties are payable in the Stacks token
-([STX](https://www.coingecko.com/en/coins/stacks)) for accepted, high-quality
-submissions.
+The Stacks Foundation has partnered with [ImmuneFi][1] to reward honest
+researchers who find and responsibly disclose security vulnerabilities in our
+critical code. Bounties are payable in the Stacks token ([STX][2]) for accepted,
+high-quality submissions.
+
+Learn more here: [https://bounty.stacks.org][3].
```

Comment:
It seems the immunefi page for Stacks doesn't mention sBTC (neither in the description nor in the assets in scope).

Reply:
Thanks! sBTC will have its own bounty, but for the time being I'll make sure the one from Stacks covers it.